Documentation/ProjectFile/prj/processes/process/TWOPHASE_FLOW_PP/material_property/porous_medium/porous_medium/a_id.md

```diff
@@ -0,0 +1,3 @@
+It specifies the material ID of a porous medium. The material IDs of elements
+are given in mesh data. If a mesh data is given in vtu format, the scalar
+variable name of MaterialIDs leads the element-wise material ID data.
```

Comment:
the name of the scalar variable ...
leads???? the element-wise material ID data